coral-hive/src/test/java/com/linkedin/coral/hive/hive2rel/HiveToRelConverterTest.java

```diff
@@ -633,6 +633,17 @@ public void testNameSakeColumnNamesShouldGetUniqueIdentifiers() {
     assertEquals(generated, expected);
   }
 
+  @Test
+  public void testCaseInsensitiveFieldAccess() {
```

Comment:
I'm not sure if I'm doing anything wrong, but it seems this test passes even without your coral-hive main source code changes. Does this imply this unit test is not well designed?